scripts/create-seed-fixture

```diff
@@ -65,30 +54,29 @@ git checkout feature/branch
 git checkout orphaned-branch
 git checkout main
 
-RAD_HOME=$ALICE_RAD_HOME rad init --name "source-browsing" --description "Git repository for source browsing tests" --default-branch "main" --no-confirm
-
-sleep 10
-RAD_HOME=$ALICE_RAD_HOME rad push --seed 0.0.0.0:8778 --all --sync
-PROJECT_URN=$(rad .)
+RAD_HOME=$ALICE_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad init --name "source-browsing" --description "Git repository for source browsing tests" --default-branch "main" --no-confirm
+#RAD_HOME=$ALICE_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad push --seed $SEED --all --sync
+PROJECT_ID=$(RAD_HOME=$ALICE_RAD_HOME RAD_PASSPHRASE=$PASSPHRASE rad .)
 
 
-GIT_AUTHOR_NAME="Bob Belcher"
-GIT_AUTHOR_EMAIL="bob@radicle.xyz"
-GIT_COMMITTER_NAME="Bob Belcher"
-GIT_COMMITTER_EMAIL="bob@radicle.xyz"
+#GIT_AUTHOR_NAME="Bob Belcher"
```

Comment:
Could we add a `TODO` here, so we don't forget to uncomment this?

Reply:
We won't, this is too critical to forget about

Reply:
So open an issue?

Reply:
You mean clean the commented out part and open an issue for later?

Reply:
I'll take care of this.